src/GitReleaseManager.Core/ReleaseNotes/ReleaseNotesBuilder.cs

```diff
@@ -60,14 +60,16 @@ public async Task<string> BuildReleaseNotesAsync(string user, string repository,
 
             if (issues.Count == 0)
             {
-                var logMessage = string.Format("No closed issues have been found for milestone {0}, or all assigned issues are meant to be excluded from release notes, aborting creation of release.", _milestoneTitle);
+                var logMessage = string.Format(CultureInfo.InvariantCulture, "No closed issues have been found for milestone {0}, or all assigned issues are meant to be excluded from release notes, aborting creation of release.", _milestoneTitle);
```

Comment:
Is this outputted to the user, intentionally?
If so this should use `CultureInfo.CurrentUiCulture`, not the invariant culture.

```suggestion
                var logMessage = string.Format(CultureInfo.InvariantCulture, "No closed issues have been found for milestone {0}, or all assigned issues are meant to be excluded from release notes, aborting release creation.", _milestoneTitle);
```